src/test/java/org/opentripplanner/graph_builder/module/FakeGraph.java

```diff
@@ -67,7 +67,7 @@ public static void addRegularStopGrid(Graph g) {
       for (double lon = -83.1341; lon < -82.8646; lon += 0.005) {
         String id = Integer.toString(count++);
         RegularStop stop = TransitModelForTest.stop(id).withCoordinate(lat, lon).build();
-        new TransitStopVertexBuilder().withGraph(g).withStop(stop).build();
+        new TransitStopVertexBuilder().withStop(stop).build();
```

Comment:
You're right. I was fooled by the tests passing but that only happened because the list of vertices in the graph was empty. I've added the vertices back to the graph in 92142cffcee4aec28e85f7251b5dd60f70a2a480 and added a test to make sure that there really are vertices in the graph.